src/app/pages/observations/observation-detail.component.ts

```diff
@@ -704,6 +724,25 @@ export class ObservationDetailComponent {
     });
   }
 
+  public isEvidenceTypeOnReport(type: string): boolean {
+    return type === 'Evidence presented in the report';
+  }
+
+  public onChangeEvidenceType(type: string): void {
+    if (this.isEvidenceTypeOnReport(type)) {
+      if (this.documents.length) {
+        this.translateService.get('observation.evidence.filesDeleteNotification')
+          .subscribe((phrase: string) => window.alert(phrase));
```

Comment:
Could you use `window.confirm` instead? That means that the user can say “no” and we would revert to the previous evidence type.